Swapped Collection.all {} for Collection.each {}

rspieldenner opened the pull request:
From my testing trying to use this plugin with gradle 3.0:
No signature of method: java.util.ArrayList.all() is applicable for argument types: (nl.javadude.gradle.plugins.license.LicensePlugin$_configureSourceSetRule_closure10$_closure48)

hierynomus commented:
I've reverted your commit, and changed the type to DomainObjectCollection of sourcesets. The `.all({...})` is a method of `DomainObjectCollection` that ensures that all current and later added objects get the same closure applied. The `.each({...})` method does not ensure that.

rspieldenner commented:
Sounds good, thanks.

On Fri, Aug 19, 2016 at 3:18 AM Jeroen van Erp <notifications@github.com>
wrote:

> I've reverted your commit, and changed the type to DomainObjectCollection
> of sourcesets. The .all({...}) is a method of DomainObjectCollection that
> ensures that all current and later added objects get the same closure
> applied. The .each({...}) method does not ensure that.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hierynomus/license-gradle-plugin/pull/119#issuecomment-240982167>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAvvFjaxJ12HDY0HHeYJ5u3N78ixK5fVks5qhYLpgaJpZM4Jm2yG>
> .
>
